packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -114,6 +123,11 @@ export class TestRunnerService {
 		const executionId = await this.workflowRunner.run(data);
 		assert(executionId);
 
+		// Listen to the abort signal to stop the execution in case test run is cancelled
+		abortSignal.addEventListener('abort', () => {
+			this.activeExecutions.stopExecution(executionId);
```

Comment:
This will cause the `postExecutePromise` to throw `ExecutionCancelledError`. I don't see us catching that anywhere. Maybe we should also throw the same error within this service instead of returning and handling that centrally in `runTest`

Reply:
Thanks, I've added a try catch block in the `runTest` to capture those. Unexpected perk - it also works out of the box in case user cancels any of the evaluation-related executions from the UI.